tests/regression/test_expressions.py

```diff
@@ -33,6 +33,12 @@ def vfs(request, vcg1, cg1vcg1):
             'cg1vcg1[1]': cg1vcg1[1]}[request.param]
 
 
+@pytest.fixture(scope='module', params=['tcg1'])
+def tfs(request, tcg1):
+    """A parametrized fixture for tensor function spaces."""
+    return {'tcg1': tcg1}[request.param]
```

Comment:
FWIW, given that there's only one tensor space, you could have written:
```
@pytest.fixture(scope='module')
def tfs(tcg1):
    return tcg1
```

But it's not a show-stopper :p